.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I added this file to make f5 debugging of the tool as it is run in the pipeline quite simple to do.